### PR TITLE
Add "Entering TimeSeries Data" document

### PR DESCRIPTION
**Pull request by mal117**

### CODE REVIEW

File: src/.vuepress/sidebar/V1.3.x/zh.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ export const zhSidebar = {
       prefix: 'Basic-Concept/',
       // children: 'structure',
       children: [
+        { text: '走进时序数据', link: 'Entering_TimeSeries_Data' },
```
Comment (StefanieZhao7) [inline src/.vuepress/sidebar/V1.3.x/zh.ts:57]: Navigating Time Series Data

File: src/UserGuide/Master/Basic-Concept/Entering_TimeSeries_Data.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+# Entering Time Series Data
+
+## What Is Temporal Data？
```
Comment (StefanieZhao7) [inline src/UserGuide/Master/Basic-Concept/Entering_TimeSeries_Data.md:23]: Time Series Data

Comment (StefanieZhao7) [reply src/UserGuide/Master/Basic-Concept/Entering_TimeSeries_Data.md:23]: 英文版时序数据的翻译更新

File: src/UserGuide/latest/Basic-Concept/Navigating_Time_Series_Data.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+![](https://alioss.timecho.com/docs/img/20240505154735.png)
+
+Generally speaking, we refer to each collection point as a measurement point (also known as a physical quantity, time series, timeline, signal quantity, indicator, measurement value, etc.). Each measurement point continuously collects new data information over time, forming a time series. In the form of a table, each time series is a table formed by two columns: time and value; In a graphical way, each time series is a trend chart formed over time, which can also be vividly referred to as the device's electrocardiogram.
+
+![](https://alioss.timecho.com/docs/img/20240505154843.png)
+
+The massive time series data generated by sensors is the foundation of digital transformation in various industries, so our modeling of time series data mainly focuses on equipment and sensors.
+
+## What are the key concepts in time series data?
```
Comment (StefanieZhao7) [inline src/UserGuide/latest/Basic-Concept/Navigating_Time_Series_Data.md:35]: Key Concepts of Time Series Data

**Review by Beyyes — APPROVED** [review]

**Pull request merged**